index.js

```diff
@@ -0,0 +1,52 @@
+
+const numberElements = 5;
+
+const getImageShell = () => {
+  const imgWrap = document.querySelector('.wrap-for-copy').cloneNode(true);
+  document.querySelector('.wrap-for-copy').remove();
+  imgWrap.classList.remove('wrap-for-copy');
+  return imgWrap;
+}
+
+const imgForCopy = getImageShell();
+
+const replaceImage = (serverArray, neededImg) => {
```

Comment:
showPosts(posts) {
// TODO: append new elements to the DOM tree
// return void
}